.github/workflows/npm-publish.yaml

```diff
@@ -0,0 +1,30 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
```

Comment:
will this build? looking at the docs on npm ci it's unclear to me whether it's just npm install or also npm run build

Reply:
looks like it just installs
https://github.com/coilhq/receipt-verifier/runs/690984144?check_suite_focus=true
I was copying from here which isn't for typescript
https://github.com/actions/starter-workflows/blob/master/ci/npm-publish.yml
updated https://github.com/coilhq/receipt-verifier/pull/5/commits/efda2b150853e46e2910488b8032d64aed6c35ea